### PR TITLE
Style B: Fix category font family

### LINKED ISSUE #264: Style B: Not using correct typeface for categories
image

It appears to be using the font for the categories from the default style.

### PR DESCRIPTION
**Pull request by laurelfulford**
### All Submissions:

* [x] Have you followed the [Newspack Contributing guideline](https://github.com/Automattic/newspack-theme/blob/master/.github/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) and [VIP Go coding standards](https://vip.wordpress.com/documentation/vip-go/code-review-blockers-warnings-notices/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?

### Changes proposed in this Pull Request:

For some reason, the category font face in single posts started falling back to the default header typeface when I used Style B and C.

The cause seemed to be something about how the Sass was compiling; the fix I found was to declare the original properties the `! default` value, so they basically act as the fallback, and are only used if the `$font__heading` and `$font_body` variables aren't otherwise set.

**Before:**

image

**After:** 

image

I'm not sure if this is happening for anyone else, or if it's just a weird fluke on my machine, but this should at least harden the setup we have in place against weirdness like this.

Closes #264 .

### How to test the changes in this Pull Request:

1. Navigate to Customize > Style Pack and set it to Style 1 (Style B).
2. View a single post; confirm that the category font family is not using 'Fira Sans Condensed'.
3. Apply the PR and run `npm run build`.
4. Refresh the single page; see the correct font family.

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?

### CODE REVIEW

File: sass/variables-site/_fonts.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Font and typographic variables
 
-$font__body: Georgia, Garamond, "Times New Roman", serif;
-$font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif;
+$font__body: Georgia, Garamond, "Times New Roman", serif !default;
+ $font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif !default;
```
Comment (jeffersonrabb) [inline sass/variables-site/_fonts.scss]: Extra space at the start of this, I assume accidental?
```suggestion
$font__heading: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue", sans-serif !default;
```

**Comment by laurelfulford** [comment]
> This seems fine, the only testing question I had is the screenshots appear to be Style Pack 1 (A), while the testing instructions reference B & C?

Ugh, I gotta change those names -- they are beyond illogical in this state. The names that are there now line up with the mockups like this: 

Default = Style A
Style 1 = Style B
Style 2 = Style C... etc.

I noticed this issue in both Style 1 (B) and Style 2 (C).

File: sass/variables-site/_fonts.scss (hunk 1 above)
Comment (laurelfulford) [reply sass/variables-site/_fonts.scss]: Ah shoot! Yes, entirely accidental - it has been removed in a82df1f.

**Comment by laurelfulford** [comment]
Thanks!

**Pull request merged**